services/database.ts

```diff
@@ -101,7 +110,7 @@ export function pairStudents(studentPairs, teacherSocket: Socket) {
   for (const [tempStudent1, tempStudent2] of studentPairs) {
     const student1 = getStudent(tempStudent1.socketId);
     const student2 = getStudent(tempStudent2.socketId);
-    const chatId = student1.socket.id + '#' + student2.socket.id;
+    const chatId = (student1.socket.id + '#' + student2.socket.id) as ChatId;
```

Comment:
This is super minor. I think its cleaner without the parenthesis. 
```suggestion
    const chatId = `${student1.socket.id}#${student2.socket.id}` as ChatId;
```